Contiki project story

mybits opened the pull request:
PT design: https://www.pivotaltracker.com/story/show/104540826
PT dev: https://www.pivotaltracker.com/story/show/104541518

mybits committed:
Add basic content and structure based on the current Contiki website and new SH24 page
to be modified when the Contiki project story design will be ready

mybits committed:
Content changes in contiki project story, including changing the content order according to the design

mybits committed:
Contiki header title should be 100% width

mybits committed:
Add undordered bullet list for representing offshoring partnership data in Contiki project

mybits committed:
Change Contiki statistics section background colour to navy and text to be white

mybits committed:
Objectives vs outcomes section in Contiki project to be navy and the list to be bulleted

mybits committed:
Team section changes:
- add missing caricatures
- all caricatures height cropped to get rid off white space at the bottom
- change css classes to be more generic and used acrossed all project stories

mybits committed:
Copy changes in Contiki project title

mybits committed:
Implement copy changes for the Contiki project page

mybits committed:
Change stats background colour to match new design

mybits committed:
Modify objectives vs outcomes section because of copy changes

mybits committed:
Copy changes in quotes in Contiki project

mybits committed:
Add some of missing images and delete redundand image placeholders

mybits committed:
Change statistics section to follow the current design:
- divide stats into two sections instead of one
- one section to have one stats data and the second one to represent two columns of sats
- increase percentage numbers

mybits committed:
Fix mobile Contiki team section so caricatures are displayed in one row

mybits committed:
Replace placeholder external links with real ones

mybits committed:
Replace placeholders with the related articles tiles content

mybits committed:
Add font-awesome icons assets

mybits committed:
Remove last placeholder image, we might add the last missing asset image later

mybits committed:
Add arrow icons to statistics numbers

mybits committed:
Fix social icons class that were missing after rebasing from staging

tomsabin commented on source/assets/stylesheets/_variables.scss:
```diff
@@ -11,6 +11,7 @@ $orange: #FDAD2B;
 $yellow: #FED336;
 
 $black: #000000;
+$very-dark-gray: #333333;
```
Do we definitely need this colour? Here's `$very-dark-gray` and `$dark-gray` together. 

![screen shot 2015-10-12 at 10 13 14](https://cloud.githubusercontent.com/assets/885223/10424799/2ee3b4c6-70ca-11e5-9cb8-d004ef73f63a.png)

Just trying to prevent variable names like `gray` becoming out of hand and to something like

```
$grey: #595959;
$darkish-grey: #444444;
$dark-grey: #363636;
$darker-grey: #303030;
$darkest-grey: #292929;
$darkerer-grey: #191919;
$darkerest-grey: #111111;
```

mybits replied:
Hah, yeah I wasn't very happy about that name neither; changed to lighter black here: ec28286

tomsabin commented on source/assets/stylesheets/styles/_bitesize.scss:
```diff
@@ -82,14 +115,90 @@
+  padding-top: 20px;
+  text-align: center;
+
+  i {
```
On first glance I thought this was styling for the `italic` tag, but realised it was actually for FontAwesome. Perhaps we can add a new class to avoid confusion and better describe what it is here we are styling.

mybits replied:
Oh good point, I added custom class here: ec28286

mybits committed:
Using markdown helper method instead of middleman one to render markdown content in Contiki story
(for some reasons middleman markdown method stopped rendered the content - possibly a middleman issue)

tomsabin commented on source/assets/stylesheets/styles/_project_team.scss:
```diff
@@ -27,13 +27,23 @@
+.project-team__headshot-container {
   display: block;
   float: left;
   width: 20%;
```
These widths should be extracted out to their own classes: we have `.project-team__headshot-container` as our base class, but at the moment it is 5 people by default. We will be overwriting this each time because we know that team sizes will vary for each project.

mybits replied:
Thanks! Added new class for different container width here: 3ce86c3

tomsabin commented on source/project-stories/contiki.erb:
```diff
@@ -0,0 +1,394 @@
+      link: ""
+    - short_name: "gavin"
+      full_name: "Gavin van Lelyveld"
+      link: ""
```
For people that no longer at unboxed should either have their links removed (so the page doesn't just link back to itself) or retain the link to the people page and we will handle alumni employees / "not found" pages there.

mybits replied:
Good point, I removed the links from ppl that left the company: 3ce86c3

mybits committed:
Add more spacing to partnership results list

tomsabin commented:
For SEO sakes, we should rename the file from `contiki` to `contiki-holidays` to retain the URL

tomsabin commented on source/project-stories/contiki.erb:
```diff
@@ -0,0 +1,394 @@
+        <% current_page.data.objectives_outcomes.objective_smashed.each do |objective| %>
+          <li class="bitesize__list-item--white">
+            <%= markdown objective %>
+            <span>Objective smashed!</span>
```
Smashing..

![](https://media.giphy.com/media/ZNHYND1c5ihYQ/giphy.gif)

mybits committed:
Remove a very-dark-gray color variable and use lighter black instead,
add custom class to a FontAwesome icon for a better readability

mybits committed:
Team section changes:
- extract width from the headshot container to a separate class
- people that have left the company are not linked

mybits committed:
Rename contiki page to contiki-holidays to keep the same url as on the current website

mybits committed:
Replace contiki header image

mybits commented:
Thanks for all the comments @tomsabin - all addressed in separate commits. Also the there's a new header image replaced.

tomsabin commented on source/assets/stylesheets/_variables.scss:
```diff
@@ -58,6 +58,10 @@ $white-tile-background-color: $white;
 $white-tile-type-color: $light-purple;
 
+// Contiki
+$bitesize-dark-gray-background-color: lighten($black, 20%);
```
:fireworks:

tomsabin commented on source/project-stories/contiki-holidays.erb:
```diff
@@ -0,0 +1,399 @@
+    - short_name: "heidi"
+      full_name: "Heidi Geisler"
+      link: "people/heidi-geisler"
+  people_archived:
```
I think it might be best to handle these archived people on the people page, rather than a per project basis. It saves us having to update all the current project stories if somebody leaves, instead leaving this responsibility to be done elsewhere (on the people page itself). So perhaps we could have this list including Ben and Gavin with them linking to their own people page (despite it being a 404 - it's a currently problem which we will be addressing in the future).

mybits replied:
Yeah, that thought has passed my mind too (updating stories when ppl leave). Okey then changing to link them to 404! :)

tomsabin commented on source/project-stories/contiki-holidays.erb:
```diff
@@ -0,0 +1,399 @@
+    </p>
+  </blockquote>
+
+  <div class="project-team">
```
This should probably be a `section`, I realise this is also the same case for SH:24 that I overlooked

mybits replied:
Ok, will fix in all 3 project stories

mybits committed:
Ex-unboxders will be linked to 404 in the end, handled by a people page

mybits committed:
Add more padding to bloquotes so it the quote sign is aligned with the text in tiles

mybits committed:
Change divs to section elements

tomsabin commented on source/project-stories/contiki-holidays.erb:
```diff
@@ -0,0 +1,392 @@
+      link: "/people/richard-archer"
+    - short_name: "heidi"
+      full_name: "Heidi Geisler"
+      link: "people/heidi-geisler"
```
These need the `/` in front of them otherwise they link to `http://localhost:4567/project-stories/people/heidi-geisler`

![](http://www.hahastop.com/pictures/Oh_Noes167.jpg)

mybits replied:
Aaaaaaaa!!!! fixed: ab9d6e2

mybits committed:
Fix team members urls

tomsabin commented:
LGTM :dart:
